client/blocks/site/style.scss

```diff
@@ -118,6 +119,39 @@
 	}
 }
 
+.site__link-hint {
+	flex: 0 1 auto;
+	position: relative;
+
+	// from layout/sidebar/styles.scss a.sidebar-button
+	box-sizing: border-box;
+	white-space: nowrap;
+	overflow: hidden;
+	padding: 2px 8px 3px 8px;
+	height: 24px;
+	margin: 11px 8px 0 0;
```

Comment:
are you using top margin to position things? `align-items: center` from the flex parent should do it.